templates/ovndbcluster/bin/cleanup.sh

```diff
@@ -44,3 +44,11 @@ if [[ "$(hostname)" != "{{ .SERVICE_NAME }}-0" ]]; then
     # now that we left, the database file is no longer valid
     cleanup_db_file
 fi
+
+# Stop the DB server gracefully; this will also end the pod running script.
+# By executing the stop command in the backgroung, we guarantee to exit
+# this script and not fail with "FailedPreStopHook", while the database is
+# stopped and the pod exits.
+if [[ "$(hostname)" == "{{ .SERVICE_NAME }}-0" ]]; then
```

Comment:
Could you please explain why only the -0 member is given a chance to gracefully exit? (And the rest will get SIGKILLs from kubelet, I think.)

Reply:
That's probably a leftover of my previous tests. I initially thought that this script was stopping the DB server and deleting the DB file, but this is wrong: this script is just stopping the ovsdb daemon, nothing else. This should apply to any DB server.